src/reducer/reducer.js

```diff
@@ -0,0 +1,54 @@
+import { BUY_ITEM, REMOVE_FEATURE } from "../actions/actions";
+
+export const initialState = {
+  additionalPrice: 0,
+  car: {
+    price: 26395,
+    name: "2019 Ford Mustang",
+    image:
+      "https://cdn.motor1.com/images/mgl/0AN2V/s1/2019-ford-mustang-bullitt.jpg",
+    features: []
+  },
+  additionalFeatures: [
+    { id: 1, name: "V-6 engine", price: 1500 },
+    { id: 2, name: "Racing detail package", price: 1500 },
+    { id: 3, name: "Premium sound system", price: 500 },
+    { id: 4, name: "Rear spoiler", price: 250 }
+  ]
+};
+
+export function reducer(state = initialState, action) {
+  switch (action.type) {
+    case BUY_ITEM:
+      return {
+        ...state,
+        additionalPrice: (state.additionalPrice += action.payload.price),
```

Comment:
I don't think we need the `()` for the calculation as it's already being scoped via the `:` and `,`

In addition, what does `+=` do?

Reply:
makes sense.

+= would add the additional price into the current price held in state